utils/dev-tool/src/setup.rs

```diff
@@ -0,0 +1,34 @@
+use std::env;
+use std::process::{Command, Stdio};
+
+const NEEDED_COMMANDS: [&str; 4] = ["cargo", "cbindgen", "swift", "cargo-ndk"];
+const NEEDED_ENV_VARS: [&str; 2] = ["ANDROID_HOME", "ANDROID_NDK"];
+
+pub fn verify_ci_environment() {
+    for command in NEEDED_COMMANDS {
+        verify_command_exists(command);
+    }
+
+    for env_var in NEEDED_ENV_VARS {
+        verify_env_var_exists(env_var);
+    }
+}
+
+pub fn verify_command_exists(command: &str) {
+    let command_result = Command::new("which")
+        .arg(command)
+        .stdout(Stdio::null())
+        .stderr(Stdio::null())
+        .status()
+        .unwrap();
+
+    if !command_result.success() {
+        panic!("You do not have '{}'", command);
+    }
+}
+
+pub fn verify_env_var_exists(env_var: &str) {
```

Comment:
Neither of these functions need to be `pub`, and so imo they don't need to be functions if they're used in one spot in this small file.